package.json

```diff
@@ -52,11 +54,13 @@
     "request": "^2.87.0",
     "sequelize": "^5.11.0",
     "sequelize-cli": "^5.5.0",
+    "slug": "^1.1.0",
     "swagger-jsdoc": "^3.3.0",
     "swagger-ui-express": "^4.0.7",
     "underscore": "^1.9.1",
     "winston": "^3.2.1",
-    "zombie": "^6.1.4"
+    "zombie": "^6.1.4",
```

Comment:
this too

Reply:
I have one package locally. This was done during rebase. I just accepted both changes and removed some of the codebases I already had. Maybe that's why this is happening.